pulp_container/app/viewsets.py

```diff
@@ -1160,6 +1160,12 @@ class ContainerNamespaceViewSet(
                 "effect": "allow",
                 "condition": "has_model_perms:container.add_containernamespace",
             },
+            {
+                "action": ["create"],
```

Comment:
we need to talk through what needs to be done whenever access policy changes, probably nothing for additional changes?

Reply:
Well, i believe, as long as the access policy is marked "unchanged by the user" we would just overwrite it. But yes the code for that is not yet in place.